SCloudRandomizer/MainViewController.m

```diff
@@ -397,6 +415,16 @@ - (void)setupUI:(Track *)track {
     });
 }
 
+- (void)updateTime:(NSTimer *)timer {
+    self.scrubber.value = self.scAudioStream.playPosition;
+    self.lblCurrentTime.text = [Utility formatDuration:self.scrubber.value];
+    
+    if (self.prepToPlayHud != nil  && !self.isPlayingForFirstTime) {
```

Comment:
This is due to the weirdness in SCAudioStream where it could just skip to next song while loading current. Its weird.